com.avaloq.tools.ddk.xtext.ide/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,13 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: com.avaloq.tools.ddk.xtext.ide
+Bundle-SymbolicName: com.avaloq.tools.ddk.xtext.ide;singleton:=true
+Bundle-Version: 8.3.0.qualifier
+Bundle-Vendor: Avaloq Evolution AG
```

Comment:
Other manifest have the line
Bundle-ActivationPolicy: lazy
shall we not have it here as well?